server/server.js

```diff
@@ -26,211 +41,226 @@ const apiKey = process.env.API_KEY;
 const client = yelp.client(apiKey);
 //S.O stuff stops here
 
-/* 'use strict';
-
-const yelp = require('yelp-fusion');
-const client = yelp.client('YOUR_API_KEY');
+//creates an endpoint for the route /api;
+app.get("/", (req, res) => {
+  //console.log(req.oidc.isAuthenticated());
+  //res.send(req.oidc.isAuthenticated() ? 'Logged in' : 'Logged out');
+  res.sendFile(path.join(REACT_BUILD_DIR, "index.html"));
+});
```

Comment:
This looks like the routing for the production build. Is that right?

My only note is adding routes for all other files as well. Looks good otherwise!